stable/kubernetes-dashboard/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for kubernetes-dashboard
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+image: gcr.io/google_containers/kubernetes-dashboard-amd64
+imageTag: "v1.6.0"
+imagePullPolicy: "IfNotPresent"
+
+namespace: kube-system
+
```

Comment:
github is saying this is an issue, but I don't it is is anymore. @seanknox know how to fix this?